README.md

```diff
@@ -35,7 +35,6 @@ These are the available packages in this repository.
 |--------|-----|
 | [animations](./packages/animations/) | [![pub package](https://img.shields.io/pub/v/animations.svg)](https://pub.dev/packages/animations) |
 | [fuchsia_ctl](./packages/fuchsia_ctl/) | [![pub package](https://img.shields.io/pub/v/fuchsia_ctl.svg)](https://pub.dev/packages/fuchsia_ctl) |
-| [gauge](./packages/gauge/) | [![pub package](https://img.shields.io/pub/v/gauge.svg)](https://pub.dev/packages/gauge) |
```

Comment:
We should put something on pub to say that this is no longer maintained or delete it from there or whatever the style is for deprecated packages.

Reply:
Yes, I marked it as **discontinued** on pub.dev.

Reply:
You could link to this PR as well or explain the replacement by engine 👍